daemon/cmd/kube_proxy_replacement.go

```diff
@@ -258,7 +258,6 @@ func initKubeProxyReplacementOptions() (strict bool) {
 				_, found = h["bpf_redirect_neigh"]
 			}
 			if !found {
-				log.Warnf("Re-enabling host legacy routing as bypassing needs kernel 5.10.0 or newer.")
```

Comment:
Maybe still an info msg for users to be able to determine an exact datapath configuration?

Reply:
I added cilium status dump which might be easier to introspect on a live system than old log potentially, given we also don't add any Info msg when pre-conditions don't apply, I removed it from here. We can definitely revisit though.